### PR TITLE
feat : Added product Page

### PR DESCRIPTION
**Pull request by aqsaaqeel**

### CODE REVIEW

**Review by ishapatell1 — APPROVED** [review]

File: index.css [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+.content-area{
+    display: flex;
+}
+.hero-image .hero-img{
+    position: relative;
+}
+.content-heading .hero-heading{
+    position: absolute;
+    left:10%;
+    /* right: 20%; */
```
Comment (devananth) [inline index.css]: commented lines can be removed

File: index.css [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    max-width: 12rem;
+    max-height: 12rem;
+    border-radius: 0.3rem;
+}
+.photo-comp{
+    display:flex;
+    justify-content: flex-end;
+}
+.fa-heart{
+    color: red;
```
Comment (devananth) [inline index.css]: `css variables` can be used and using **HEX** codes or **rgba()** is recommended. Don't use **direct color names**

File: index.css [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+}
+.photo-card-container .basic-card-container{
+    position: relative;
+    padding : 0;
+}
+.photo-card-container .card-footer{
+    /* margin-bottom: 1rem; */
```
Comment (devananth) [inline index.css]: Commented lines can be removed

File: styles/product-page.css [hunk 4]
```diff
@@ -0,0 +1,355 @@
...
+    cursor: pointer;
+}
+.buttons .button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+}
+.button-secondary{
+    background-color: #F5F5F5;
```
Comment (devananth) [inline styles/product-page.css]: **css variables** can be used.

File: styles/product-page.css [hunk 5]
```diff
@@ -0,0 +1,355 @@
...
+    color: var(--primary-color);
+}
+.button:hover{
+    background-color: white;
```
Comment (devananth) [inline styles/product-page.css]: Here too.

File: styles/product-page.css [hunk 6]
```diff
@@ -0,0 +1,355 @@
...
+    margin-top: 3rem;
+    margin-right: 1.5rem;
+}
+.photo-comp{
+    display:flex;
+    justify-content: flex-end;
+}
+.fa-heart{
+    color: red;
+}
```
Comment (devananth) [inline styles/product-page.css]: I think you have already declared this **class** in **index.css** . Re-check it once

**Review by devananth — CHANGES_REQUESTED** [review]
Good work !

I have few suggestions: 

    1. Try using css variables for colors. you have forgot to do so in some places.
    2. Few classes are repeated in different css files. Re-check it once.

**Review by spraveenofficial — APPROVED** [review]

**Pull request merged**